src/main/java/io/fabric8/vertx/maven/plugin/utils/MojoUtils.java

```diff
@@ -29,30 +29,37 @@
 import java.util.List;
 import java.util.Optional;
 
-import static org.twdata.maven.mojoexecutor.MojoExecutor.configuration;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.element;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.executeMojo;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.executionEnvironment;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.goal;
-import static org.twdata.maven.mojoexecutor.MojoExecutor.plugin;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.*;
 
 /**
  * @author kameshs
  */
 public class MojoUtils {
 
+    /*===  Plugin Keys ====*/
+
     public static final String JAR_PLUGIN_KEY = "org.apache.maven.plugins:maven-jar-plugin";
+    public static final String RESOURCES_PLUGIN_KEY = "org.apache.maven.plugins:maven-resources-plugin";
     public static final String VERTX_PACKAGE_PLUGIN_KEY = "io.fabric8:vertx-maven-plugin";
 
+    /*===  Plugins ====*/
 
     public static final String G_MAVEN_JAR_PLUGIN = "org.apache.maven.plugins";
     public static final String A_MAVEN_JAR_PLUGIN = "maven-jar-plugin";
     public static final String V_MAVEN_JAR_PLUGIN = "3.0.2";
 
+    public static final String G_MAVEN_RESOURCES_PLUGIN = "org.apache.maven.plugins";
+    public static final String A_MAVEN_RESOURCES_PLUGIN = "maven-resources-plugin";
+    public static final String V_MAVEN_RESOURCES_PLUGIN = "3.0.1";
```

Comment:
Please try to avoid hardcoding version numbers into the source code. This should be taken either from the pom.xml (e.g. the same plugin version as used by the project using this plugin) or, if not there, from some properties holding only the version for default values. That way, a user always can specify the version of the plugin it wants to use (via declaring it in the pom.xml) and all default versions would be nicely in a single file.